soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -342,6 +342,19 @@ void Randomizer::LoadHintLocations(const char* spoilerFileName) {
         "Warp to&{{location}}?\x1B&%gOK&No%w\x02",
         "Warp to&{{location}}?\x1B&%gOK&No%w\x02", // TODO: German translation
         "Se téléporter vers&{{location}}?\x1B&%gOK!&Non%w\x02" });
+
+    CustomMessageManager::Instance->CreateMessage(Randomizer::hintMessageTableID, TEXT_LAKE_HYLIA_WATER_SWITCH_SIGN,
+        { TEXTBOX_TYPE_WOODEN, TEXTBOX_POS_BOTTOM,
+            "Water level control system.&Keep away!",
+            "Wasserstand Kontrollsystem&Finger weg!",
```

Comment:
Are these AI generated? If so let's add a TODO comment on them

Reply:
Ripped from 3ds rando, the french one was corrected by Purple